<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeScaleBase</class>
 <widget class="QDialog" name="SizeScaleBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Field</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scale method</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Size from</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsDoubleSpinBox" name="minSizeSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QgsDoubleSpinBox" name="maxSizeSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Values from</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsDoubleSpinBox" name="minValueSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-99999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QgsDoubleSpinBox" name="maxValueSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-99999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QToolButton" name="computeValuesButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QgsFieldExpressionWidget" name="mExpressionWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="scaleMethodComboBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QgsDoubleSpinBox" name="exponentSpinBox">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="value">
        <double>0.570000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>154</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Size when field is NULL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsDoubleSpinBox" name="nullSizeSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgsfieldexpressionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mExpressionWidget</tabstop>
  <tabstop>scaleMethodComboBox</tabstop>
  <tabstop>exponentSpinBox</tabstop>
  <tabstop>minSizeSpinBox</tabstop>
  <tabstop>maxSizeSpinBox</tabstop>
  <tabstop>minValueSpinBox</tabstop>
  <tabstop>maxValueSpinBox</tabstop>
  <tabstop>computeValuesButton</tabstop>
  <tabstop>nullSizeSpinBox</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SizeScaleBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SizeScaleBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
